src/components/sections/CaseStudyAndCoreToolsKits.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { Entries, Entry, LevelEntries } from "@/types/interfaces";
+import { NodeCard } from "../cards/Node";
+interface Props {
+  categorizedEntries: LevelEntries;
+  isToolsBlock: Boolean;
+}
+
```

Comment:
Can you put this interface into types/interface file and import it here

Reply:
It is not going to be reused anywhere, all prop interfaces are in the file we are using them, We thought we can let them stay there.